include/irr/asset/IPipelineLayout.h

```diff
@@ -43,6 +57,16 @@ struct SPushConstantRange
     }
 };
 
+//! Interface class for pipeline layouts
+/*
+    Pipeline layout stores all the state (options) common to multiple
```

Comment:
not options, bindings and set numbers of descriptors as well as the descriptor types

Reply:
adjusted in following commit